src/entity/active_enum.rs

```diff
@@ -396,14 +432,14 @@ mod tests {
         }
 
         test_num_value_uint!(U8, "u8", "TinyInteger", TinyInteger);
-        test_num_value_uint!(U16, "u16", "SmallInteger", SmallInteger);
+        // test_num_value_uint!(U16, "u16", "SmallInteger", SmallInteger);
```

Comment:
@tyt2y3 @billy1624 Can you please help me out with this one? I wrecked my brain a lot already but couldn't figure it out.
1. I'm note sure why these use the Integer defs and not Unsigned defs (`TinyInteger` instead of `TinyUnsigned` for example.
2. I just cannot figure out why these tests worked before, given that `u16` doesn't implement sqlx's `Decode` trait and `u64` doesn't implement sqlx's `PgHasArray` trait, so I cannot add a manual `TryGetable` implementation for either of them.